README.md

```diff
@@ -63,6 +63,8 @@ Run `sudo pip3 install python-bidi google-api-python-client google-auth-httplib2
                         max number of events to query Google's API for each of your calendars. Increase this number if you have lot of events in your google calendar
   --today, -d           print only today events
   --no-event-text TEXT  text to display when there are no events
+  --started-hide-minutes MINUTES, -s MINUTES
```

Comment:
rename this argument `time-to-show-event` or `hide-event-after`\
the current name is unclear

Reply:
Thanks @rosenpin, I like `hide-event-after`, but the shortcut won't be mnemonic. Would it be OK if it doesn't have a shortcut?

Reply:
that's fine